src/connectors/articleService.ts

```diff
@@ -139,13 +140,20 @@ export class ArticleService extends BaseService {
 
     const bundleInfo = {
       title,
-      author: { userName, displayName },
+      author: {
+        name: displayName,
+        link: {
+          text: `${displayName} (@${userName})`,
+          url: new URL(`/@${userName}`, environment.siteDomain).href,
+        },
+      },
+      from: {
+        text: 'Matters',
+        url: environment.siteDomain,
+      },
       summary,
-      summaryCustomized,
```

Comment:
I updated [the logic](https://github.com/thematters/matters-server/pull/2007/files#diff-6775d7dc5f0198959c6fe0e20d1aeff1620ba32b75bb328a09c7d042230da5c9R180-R185) so it only pass in summary when it is customized or when it will be encrypted